api/v1beta1/glance_types.go

```diff
@@ -69,10 +69,6 @@ type GlanceSpec struct {
 	// actual action pod gets started with sleep infinity
 	Debug GlanceDebug `json:"debug,omitempty"`
 
-	// +kubebuilder:validation:Optional
-	// CephBackend - The ceph Backend structure with all the parameters
-	CephBackend ceph.Backend `json:"cephBackend,omitempty"`
```

Comment:
Yes, let's remove this instead of trying to make them co-exist. We'll switch to the new one.